programs/scope/src/utils/math.rs

```diff
@@ -4,8 +4,12 @@ use decimal_wad::{
 };
 use raydium_amm_v3::libraries::U256;
 use solana_program::clock;
+#[cfg(feature = "yvaults")]
 use yvaults::utils::FULL_BPS;
 
+#[cfg(not(feature = "yvaults"))]
+const FULL_BPS: u64 = 10_000;
```

Comment:
I'm not sure they accept the change ;)

Reply:
this is only so the code compiles, we dont have access to yvaults

Reply:
I know